archive/s/scala/quick-sort.scala

```diff
@@ -0,0 +1,58 @@
+import scala.io.StdIn.readLine
+import scala.reflect.ClassTag
+
+object QuickSortSamples {
+  def main(args: Array[String]) {
+    // example of sorting integers
```

Comment:
These are great samples here! Unfortunately, we have special requirements for each type of sample program. You can see the description for QuickSort here: https://sample-programs.therenegadecoder.com/

We're actually in the process of writing test go to verify new samples, so this solution should follow the Quick Sort table if possible.

Reply:
I'm a bit lost on the home page haha. Can I get a direct link to the quick sort spec?

Also for in-development tests, will they be part of CI, so contributors can to use them on our own fork's travis?

Reply:
My bad! I thought I gave the right link: https://sample-programs.therenegadecoder.com/projects/quick-sort/

Also, yes! They're already built into Travis, but there's limited support at the moment languages other than Python and Bash.

Reply:
yeah gotcha, quick question for some clarifications:
```
$ ./quick-sort.lang "4, 5, 3, 1, 4, 2"
1, 2, 3, 4, 4, 5
```
so input/output format is a string, which is a comma-delimited list of integers?

Reply:
Yup! That's the idea.